package.json

```diff
@@ -133,8 +133,10 @@
   },
   "dependencies": {
     "@sentry/electron": "^0.8.1",
+    "@types/compare-versions": "^3.0.0",
```

Comment:
@nirinchev I just noticed this - I think the @types/* packages should be devDependencies, any reason that was not the case for this package?

Reply:
You're right - I must have fat-fingered the install command, sorry.